walk/config.go

```diff
@@ -104,6 +112,46 @@ func (cr *Configurer) Configure(c *config.Config, rel string, f *rule.File) {
 	c.Exts[walkName] = wcCopy
 }
 
+func (c *Configurer) loadBazelIgnore(repoRoot, rel string, wc *walkConfig) error {
+	// Only load .bazelignore at the root, and only once
+	if rel != "" {
+		return nil
+	}
+
+	ignorePath := path.Join(repoRoot, ".bazelignore")
+	file, err := os.Open(ignorePath)
+	if os.IsNotExist(err) {
+		return nil
+	}
+	if err != nil {
+		return fmt.Errorf(".bazelignore exists but couldn't be read: %v", err)
+	}
+	defer file.Close()
+
+	wc.excludes = append(wc.excludes, path.Join(rel, ".bazelignore"))
```

Comment:
why do we need to exclude the `.bazelignore` file specifically? Could we make do without this? Seems like a very minor details for gazelle to specifically ignore 1 single file given that `rel` is effectively empty.

Reply:
Probably works without it; another bit copied from the original